docs/en/sql-reference/statements/alter/settings-profile.md

```diff
@@ -11,7 +11,7 @@ Changes settings profiles.
 Syntax:
 
 ``` sql
-ALTER SETTINGS PROFILE [IF EXISTS] TO name1 [ON CLUSTER cluster_name1] [RENAME TO new_name1]
-        [, name2 [ON CLUSTER cluster_name2] [RENAME TO new_name2] ...]
+ALTER SETTINGS PROFILE [IF EXISTS] name1 [RENAME TO new_name |, name2 [,...]] 
```

Comment:
I see we also have a query like this in tests:
```
ALTER SETTINGS PROFILE xyz TO worker
```
Also, the same logic with "CREATE SETTINGS PROFILE" also works

Reply:
I didn't get it. 
I removed TO before the settings profile name since `ALTER SETTINGS PROFILE TO xyz TO worker` is not valid.

Reply:
https://github.com/ClickHouse/ClickHouse/commit/0a9f6b180563f63f4f56dbd98f5e9384597cd133#diff-78f2731e9b615259308bdbc2d4c62960530cce5c32f97651b74a96442d3c8186 it was added by this commit